srml/staking/src/lib.rs

```diff
@@ -699,19 +693,17 @@ impl<T: Trait> Module<T> {
 			}
 		}
 
-		// Figure out the minimum stake behind a slot.
-		let slot_stake = candidates.last().map(|i| i.1.total).unwrap_or_default();
-		<SlotStake<T>>::put(&slot_stake);
-
 		// Populate Stakers.
-		for (who, exposure) in &candidates {
-			<Stakers<T>>::insert(who, exposure);
+		for candidate in &elected_candidates {
+			<Stakers<T>>::insert(candidate.who.clone(), candidate.exposure.clone());
 		}
+
 		// Set the new validator set.
 		<session::Module<T>>::set_validators(
-			&candidates.into_iter().map(|i| i.0).collect::<Vec<_>>()
+			&elected_candidates.into_iter().map(|i| i.who).collect::<Vec<_>>()
 		);
-
+		
+		<SlotStake<T>>::put(&slot_stake);
```

Comment:
any reason why this was moved down here?

Reply:
While automatic merges probably. Will move it back up in the final merge.